Packs/Salesforce/Scripts/script-SalesforceAskUser.yml

```diff
@@ -1,92 +0,0 @@
-commonfields:
-  id: SalesforceAskUser
-  version: -1
-name: SalesforceAskUser
-script: >-
-  import time
```

Comment:
Why did you remove that yml? Wouldn't that cause issues with backward compatibility

Reply:
this wasn't done by me...